publishLocally.sh

```diff
@@ -12,6 +12,17 @@ sbt 'publishM2; publishLocal'
 (
   cd maven-java
   mvn versions:set -DnewVersion=$SDK_VERSION
+
+  (
+    # special case for parent pom, version:set only changes project sharing the same parent
+    # kalix-spring-boot-parent is special because its parent is spring-boot-starter-parent
+    cd kalix-spring-boot-parent
+    mvn versions:set -DnewVersion=$SDK_VERSION
+
+    # also needs to change kalix-sdk.version in parent pom
+    sed "s/<kalix-sdk.version>\(.*\)<\/kalix-sdk.version>/<kalix-sdk.version>$SDK_VERSION<\/kalix-sdk.version>/" pom.xml
+  )
```

Comment:
this is almost the same as in publish-maven.sh

The only difference is that one is doing local install and the other is deploy (publishing) the artifacts.